staging/src/k8s.io/apiserver/pkg/admission/plugin/resourcequota/controller.go

```diff
@@ -124,7 +124,7 @@ func NewQuotaEvaluator(quotaAccessor QuotaAccessor, ignoredResources map[schema.
 		ignoredResources: ignoredResources,
 		registry:         quotaRegistry,
 
-		queue:      workqueue.NewNamed("admission_quota_controller"),
+		queue:      workqueue.NewWithConfig(workqueue.QueueConfig{Name: "admission_quota_controller"}),
```

Comment:
Can we also remove NewNamed function as we removed all its calls?

Reply:
yes, will remove lator.